js/order.js

```diff
@@ -19,15 +19,16 @@ sandwiches.push(new Sandwich(this.bread, this.spreads, this.jams, this.toppings,
 
 
 var makeSammyButton = document.getElementById('makeSammy');
+//makeSammyButton.addEventListener("click", makeSandwich);
+
 makeSandwich = function() {
     var breadChoice = document.getElementById('bread-choice').value;
     var spreadChoice = document.getElementById('spread-choice').value;
     var jamChoice = document.getElementById('jam-choice').value;
     var extrasChoice = document.getElementById('extras-choice').value;
-    var topppingChoice = document.getElementById('toppings-choice').value;
     var grilledChoice = document.getElementById('grilled-choice').value;
     var cutChoice = document.getElementById('cut-choice').value;
-    sandwiches.push(new madeSandwich(bread, spread, jam, extras, grilled, cut));
+    sandwiches.push(new makeSandwich(breadChoice, spreadChoice, jamChoice, extrasChoice, grilledChoice, cutChoice));
```

Comment:
Hey, So instead of line 31 and I _think_  line 32 as well - do this...
in the main.js file (I'm sorry I might have forgotten this last night, just part of the challenge of working remote I guess!) I have a prototype function called `addFaveSandwich();` that adds a new sandwich to the favoriteSandwiches array in the UserAccounts constructor function object. On this line 31-32, you can just call that function. But first! You will need to specify which user account to put it in. I have a function that already has saved the user account index number (`accounts[index]`) in local storage so you just have to retrieve it, like so:  

`var currentUser = localStorage.getItem('current login'); `
(current login is the name of the key in local storage)

then put:
`currentUser.addFaveSandwich(all of your variables that hold the form values);`

That should update the accounts array and everything! Thanks!!